README.md

```diff
@@ -27,7 +27,7 @@ The Utility Connect widget can be instantiated for one of two user flows:
 
 The user flow is selected with the `config.scope` parameter described in the component API Reference below.
```

Comment:
This is no longer true.

I would replace with:

The user flow is selected when the `UtilityConnectToken` that's eventually passed into Utility Connect is initialized by the accompanying server: if a `utility_credential_id` is provided, Utility Connect will automatically be opened in 'update' mode (where the user updates their credentials), otherwise it's opened in 'create' mode.